ui/lib/replication/addon/templates/components/replication-summary.hbs

```diff
@@ -262,69 +262,90 @@
-      {{#if submit.isRunning }}
-        <LayoutLoading />
-      {{else}}
-        {{#link-to
-          "mode.index"
-          "dr"
-          class="link-plain"
-        }}
-          {{replication-mode-summary
-            mode="dr"
-            cluster=cluster
-            tagName="span"
-          }}
-        {{/link-to}}
-      {{/if}}
-    {{else}}
-      {{replication-mode-summary
-        mode="dr"
-        cluster=cluster
-        tagName="div"
-      }}
-    {{/if}}
-  </div>
-  {{#unless (and submit.isRunning (eq cluster.dr.mode "bootstrapping"))}}
-    <div class="box is-bottomless is-fullwidth is-marginless">
+  {{#unless (and cluster.dr.replicationEnabled cluster.performance.replicationEnabled)}}
+    <PageHeader as |p|>
+      <p.levelLeft>
+        <h1 class="title is-3">
+          Replication
+        </h1>
+      </p.levelLeft>
+    </PageHeader>
+  {{/unless}}
+
+  {{#if (and cluster.dr.replicationEnabled cluster.performance.replicationEnabled ) }}
+    <ReplicationPage @model={{cluster}} as |Page|>
+      <Page.header
+        @showTabs={{true}}
+      />
+      <Page.dashboard
+        @componentToRender='replication-summary-card' as |Dashboard|>
+        <Dashboard.card
+          @title="Disaster Recovery"
+        />
+        <Dashboard.card
+          @title="Performance"
+        />
+      </Page.dashboard>
+    </ReplicationPage>
+  {{else}}
+
+    <div class="box is-sideless is-fullwidth is-marginless">
       <h3 class="title is-flex-center is-5 is-marginless">
-        <Icon @size="xl" aria-hidden="true" @glyph="perf-replication" />
-        Performance
+        <Icon @size="xl" aria-hidden="true" @glyph="replication" />
+        Disaster Recovery (DR)
       </h3>
       {{#if cluster.dr.replicationEnabled}}
-        {{#link-to
-          "mode.index"
-          "performance"
-          class="link-plain"
-        }}
-          {{replication-mode-summary
-            mode="performance"
-            cluster=cluster
-            tagName="span"
+        {{#if submit.isRunning }}
+          <LayoutLoading />
+        {{else}}
+          {{#link-to
+            "mode.index"
+            "dr"
+            class="link-plain"
           }}
-        {{/link-to}}
+            {{replication-mode-summary
+              mode="dr"
+              cluster=cluster
+              tagName="span"
+            }}
+          {{/link-to}}
+        {{/if}}
       {{else}}
         {{replication-mode-summary
-          mode="performance"
+          mode="dr"
           cluster=cluster
           tagName="div"
         }}
       {{/if}}
     </div>
-  {{/unless}}
+      {{#unless (and submit.isRunning (eq cluster.dr.mode "bootstrapping"))}}
+      <div class="box is-bottomless is-fullwidth is-marginless">
+        <h3 class="title is-flex-center is-5 is-marginless">
+          <Icon @size="xl" aria-hidden="true" @glyph="perf-replication" />
+          Performance
+        </h3>
+        {{#if cluster.dr.replicationEnabled}}
```

Comment:
is this the right conditional here? it looks like it's checking for DR but then rendering links to performance 🤔

Reply:
Wow, OK this was logic I moved and pulled over (e.g. I didn't write this, I just moved it).  _However_, it does not make any sense.  This is the functionality of what shows if I remove the conditional.  Notice the first one is a clickable box, and the second one is not. I think I can safely remove this logic and just let the box be clickable, _but_ it's a little outside of the scope.  Thoughts?

![conditional](https://user-images.githubusercontent.com/6618863/83277525-6f6c8680-a18f-11ea-9f16-aff43eb188a1.gif)

Reply:
woah, that is bizarre 😳 yeah, i'd try removing the conditional and seeing if the tests still pass. additionally, i'd check to make sure the box correctly show/hides when DR is disabled & performance is enabled, when both are enabled, when both are disabled etc.